pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -103,7 +110,7 @@ func NewController(
 		taskLister:     taskInformer.Lister(),
 		resourceLister: resourceInformer.Lister(),
 	}
-	impl := controller.NewImpl(c, c.Logger, taskRunControllerName)
+	impl := controller.NewImpl(c, c.Logger, taskRunControllerName, reconciler.MustNewStatsReporter(taskRunControllerName, c.Logger))
```

Comment:
Sorry to go back and forth on this again, but the `MustNewStatsReporter` is creating a Stats reporter to collect metrics which is valid in serving, but we are not collecting any metrics in pipeline currently.

Reply:
Fixing.